x-pack/solutions/security/plugins/security_solution/server/endpoint/routes/workflow_insights/get_insights.test.ts

```diff
@@ -92,6 +92,94 @@ describe('Get Insights Route Handler', () => {
         body: [],
       });
     });
+    describe('with space awareness enabled', () => {
+      const enableSpaceAwareness = () => {
+        // @ts-expect-error write to readonly property
+        mockEndpointContext.experimentalFeatures.endpointManagementSpaceAwarenessEnabled = true;
+      };
+
+      it('should call ensureInCurrentSpace when flag is enabled', async () => {
+        enableSpaceAwareness();
+
+        const mockEnsureInCurrentSpace = jest.fn();
+        const mockFleetService = {
+          ensureInCurrentSpace: mockEnsureInCurrentSpace,
+        };
+
+        mockEndpointContext.service.getInternalFleetServices = jest
+          .fn()
+          .mockReturnValue(mockFleetService);
```

Comment:
FYI: this is unnecessary. Internal Fleeet services are already mocked via `createMockEndpointAppContext()`.

You can replace it with this:

```typescript
const mockEnsureInCurrentSpace = mockEndpointContext.service.getInternalFleetServices()
          .ensureInCurrentSpace as jest.Mock;
```

Reply:
Same comment for other tests below

Reply:
Cleaned up - dcb4ac5a95bc5144ab1f1c3a147cc1de1cb248ee